### PR TITLE
Get slack app dev environment working again

### LINKED ISSUE #356: Get the slack app dev enironment up and running again.
- [x] Minimal get the dev env working again after a looooong time
- [x] Move environment variables into keyvault
- [x] Consolidate Dockerfile`s
- [x] Remove devcontainer setup
- [x] Test debug setup
- [x] Integrate the dev setup into the monorepo dev setup
- [x] Update README

### PR DESCRIPTION
**Pull request by Trulsaa**

### CODE REVIEW

**Comment by Trulsaa** [comment]
Closes #356

File: packages/slack-app/Readme.md [hunk 1]
```diff
@@ -5,56 +5,11 @@ This app i built using [node slack sdk](https://slack.dev/node-slack-sdk/). Read
...
-```
-docker-compose -f .devcontainer/docker-compose.yaml logs -f
-```
-
-Download the latest backend api and database images
-
-```
-docker-compose -f .devcontainer/docker-compose.yaml pull
-```
+The only way we know how to develop the slack-app is by using the docker-compose setup. The dev environment needs some secrets we have chosen to store in a azure key vault. To get access to this key vault login using `az login`. You can now run `./run slack-app` from the root of the alvtime project. This command will start the necessary services and drop you into a bash shell in the slack app development service. Her you can run `npm run watch` to start the development server. Navigate to the [development slack workspace](dev-qje3117.slack.com/) and log in useng Ahre-Ketil-lillehagen-on-dev-qje3117-slack-com credentials found in our common test user keyvault. In the slack workspace you can run `/alvtime tasks` in anny channal. This will prompt a login flow. Use Ahre-Ketil's azure account to login to azure. The credentials can be found in the same keyvault.
```
Comment (palbock) [inline packages/slack-app/Readme.md]: Et par skriveleifer: "Her you can run" -> "Here you can run", "anny channal" -> "any channel"

Comment (Trulsaa) [reply packages/slack-app/Readme.md]: Done :)

**Review by palbock — APPROVED** [review]
Har ingenting å utsette på dette :)

**Review by sindrenordal — APPROVED** [review]

**Pull request merged**